substreams-helper/src/rpc.rs

```diff
@@ -0,0 +1,64 @@
+use substreams::Hex;
+use substreams_ethereum::{pb::eth as ethpb, rpc};
+use tiny_keccak::{Hasher, Keccak};
+
+#[derive(Debug)]
+pub struct RpcCallParams {
+    pub to: Vec<u8>,
+    pub method: String,
+    pub args: Vec<Vec<u8>>,
+}
+
+// Construct rpc data according to https://docs.soliditylang.org/en/develop/abi-spec.html#examples
+fn rpc_data(method: &str, args: &Vec<Vec<u8>>) -> Vec<u8> {
+    let method_sig = method_signature(method);
+    if args.len() == 0 {
+        return method_sig;
+    }
+    let mut data = Vec::with_capacity(8 + args.len() * 32);
+    data.extend(method_sig);
+    for arg in args {
+        data.extend(vec![0u8].repeat(32 - arg.len()));
+        data.extend(arg);
+    }
+    return data;
+}
+
+// "name()" -> "06fdde03"
```

Comment:
Should change this to a documentation test: https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#documentation-tests

Reply:
should be in a separate PR to set up rustdoc ci

Reply:
sounds good